float32 x
float32 y
float32 z
float32 length
float32 height
float32 depth
string class_name
